source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,60 @@
+RoboRIO2 Imaging
+================
+
+.. note:: The imaging instructions for the NI roboRIO 1.0 are :ref:`here <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`.
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the microSD card -- with the roboRIO :ref:`using the imaging tool <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.  It is recommended to use a SDHC or SDXC capacity card, though all types are expected to function correctly.
+
+SD (secure digital) cards are the oldest, least used and limited to 2GB of storage.  SDHC (high capacity) cards can store up to 32 GB of data, while SDXC (extended capacity) cards can store up to 2 terabytes (2000 GB).
+
+Operation Tips
+--------------
+
+The NI roboRIO 2.0 controller requires a fully inserted microSD card containing a valid image in order to boot and operate as intended.
+
+If the microSD card is removed while powered, the controller will hang. Once the microSD card is replaced, the controller will need to be restarted using the reset button, or be power cycled.
+
+No damage will result from microSD card removal or insertion while powered, but best practice is to perform these operations while unpowered.
```

Comment:
```suggestion
As described in the previous section, the NI roboRIO 2.0 controller requires a fully inserted microSD card. This microSD card needs to contain a valid roboRIO image to boot and operate. There are some important things to keep in mind when using the roboRIO 2.0.

If the microSD card is removed while powered, the roboRIO will hang. Once the microSD card is replaced, the roboRIO will need to be restarted using the reset button, or be power cycled.  No damage will result from microSD card removal or insertion while powered, but best practice is to perform these operations while unpowered.

.. note:: If you write files on the roboRIO 2.0 in a robot program (most teams don't do this!), then there is a chance of corrupting these files upon an unexpected microSD card removal.

```

Reply:
Cleaned it up a little bit

Reply:
As described?  It wasn't described before that it needs a fully seated card?

"valid roboRIO image" is coverd in a later paragraph.

"There are some important things to keep in mind when using the roboRIO 2.0." This sentence adds nothing.

You note is confusing. "most teams don't do this" is that teams shouldn't do this or that it is rarely done?  What is the source on this information?  If true I would like to understand more

Reply:
Have you experienced file corruption on removal of the SD Cardin a roboRIO2, Dalton?With a journaled file system it shouldn't happen. This isn't a raspberry pit with a fat32 file system. I would recommend not adding that line until it's been proven to be a problem.